WeakAuras/RegionTypes/Icon.lua

```diff
@@ -324,11 +324,19 @@ local function modify(parent, region, data)
   region.progressPrecision = data.progressPrecision;
   region.totalPrecision = data.totalPrecision;
 
-  if MSQ and not region.MSQGroup then
-    region.MSQGroup = MSQ:Group("WeakAuras", region.frameId);
-    region.MSQGroup:AddButton(button, {Icon = icon, Cooldown = cooldown});
+  if MSQ then
+    local masqueId = data.id:lower():gsub(" ", "_");
```

Comment:
This might cause surprises if a user has, e.g. `Aura_1` and `Aura 1` Installed. Is that necessary for the masque api to work?

Reply:
That has been always the case, and we can't change it anymore due to existing settings.